util/docker/exec.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+DOCKER_DIR=$(dirname "$0")
+source "${DOCKER_DIR}/../logging.sh"
+WPTDASHBOARD_DIR=${WPTDASHBOARD_DIR:-"${DOCKER_DIR}/../.."}
+
+# Execute code in docker instance:
+#
+# -u $(id -u $USER):$(id -g $USER)  Run as current user and group
+# wptd-dev-instance                 Name of instance (wptdashboard dev server)
+
+EXEC_STR=""
+for ARG in "${@}"; do
+  EXEC_STR+="\"${ARG}\" "
+done
+info "Execute-in-docker:  ${EXEC_STR}"
+docker exec -u $(id -u $USER):$(id -g $USER) wptd-dev-instance "${@}"
```

Comment:
Are these exec files necessary? You can open a shell in the containers and debug from there.

Reply:
I suppose so. It just makes other scripts that are invoking these scripts more verbose.